### DISCUSSION TITLE
[Bug]: Individual Group Filters Using Same Column Cancel Each Other Out

### DISCUSSION THREAD

**Discussion by Stilts15ak** [RafaelGB/obsidian-db-folder#619] [Q&A] [answered]
### What happened?

Group filters cancel each other out and report nothing if they are filtering the same column.

Example: Column is "Status" (Options: "On Track", "Delayed", "Archived")
Group Filter: Named "On Track" (Column "Status" contains "On Track")
Group Filter: Named "Delayed" (Column "Status" contains "Delayed")

This should filter all entries that have a status of "On Track" and "Delayed" when both group filters are selected. If you select both, no entries are displayed.

### What platform were you using?

Desktop

### Version of the plugin

2.8.4

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
the filter groups are not thinking that way.

You have to create an unique group with the OR condition and add both  filters

**Comment by Stilts15ak** [comment]
So essential you are saying that two separate filter groups targeting the same column will not function and not report anything?

Instead, you have to to include the two separate filter groups together into one filter group for it to work?

**Comment by RafaelGB** [comment, marked as answer]
Yep,

Example:

Input=> MyValue

Filter1:
Input is MyValue

(AND) Between filters there is an AND condition

Filter2:
Input is AnotherValue

Result => False (Input can not be two values at the same time)

---

Input=> MyValue

Filter1:
Input is MyValue
OR
Input is AnotherValue


Result => True

**Comment by Stilts15ak** [comment]
Thank you. This is understood.

Would like to recommend for future, if possible, if the use of filter groups in the initial inquiry could be explored.

**Reply by RafaelGB** [reply]
I don't fully understand what you mean about the initial inquiry

**Reply by Stilts15ak** [reply]
Apologies. By initial inquiry meaning that you cannot use more than one filter group that targets the same column at the same time.

Example:
Having one filter for the "Status" column that chooses "On Track" and then a completely separate group filter that chooses "Delayed". Then you can filter two (_or more_) choices at the same time within the same column, while also having the ability to filter them independently when needed.

I don't know if there is a limitation with programming or not. However, I think this could be a powerful capability when trying to filter data.

**Reply by RafaelGB** [reply]
I got it now.

You can nest group filters if you want to apply more than one with the same column. the limitation will be the quick filter Enable/disable function (internally inside the menu you can disable nested groups too)

The "and" condition between separated groups is a must

**Reply by Stilts15ak** [reply]
Yes. It looks like it works. However, it only works with the "or" condition.

The quick filter method is more where I was getting at. So that the filter group only has one condition set to it for the column.
